modules/configuring-firewall.adoc

```diff
@@ -38,9 +38,13 @@ If your environment has a dedicated load balancer in front of your {product-titl
 |443
 |Provides core container images
 
-|`access.redhat.com` ^[1]^
+|`access.redhat.com`
 |443
-|Hosts all the container images that are stored on the Red Hat Ecosytem Catalog, including core container images.
+|Hosts all the container images that are stored on the Red Hat Ecosystem Catalog, including core container images. In a firewall environment, ensure this resource is on the allowlist, because it hosts a signature store that a container client requires for verifying images when pulling them from `registry.access.redhat.com`.
```

Comment:
I don't think there's a preview available yet to verify but these look switched, i.e. this (`Hosts all the container images`) is the description for registry.access.redhat.com